src/sentry/runner/commands/cleanup.py

```diff
@@ -38,6 +38,7 @@ def multiprocess_worker(task_queue):
     # Configure within each Process
     import logging
 
+    from sentry import deletions
```

Comment:
I only needed to update an import in this file but there were `UnboundVariable` errors in this file so I had to make some changes to get rid of them. I'm not sure how performant these changes are, or so I'd appreciate someone taking a deeper look at whether or not this file was changed properly.